### DISCUSSION TITLE
Bought 4 modules but 1 is not working at all

### DISCUSSION THREAD

**Discussion by Wunderlag** [revk/xESP32-Faikout#584] [Q&A]
Hi, i just bought 4 Faikins and i can't seem to get 1 of them running... switched around all of them to the same Daikin and the troublemaker is not doing anything, no access point and the LEDs don't turn on.

Is there a way to troubleshoot what's going on?

**Comment by revk** [comment]
Well, obviously swapping with other ones to confirm it is the Faikin and not the air-con or lead.

If the Faikin won't even light up, but others work on the same unit / lead / connectors, then it is likely faulty and should be returned.
